### DISCUSSION TITLE
Successful Installation Test

### DISCUSSION THREAD

**Discussion by dSofikitis** [gstavrinos/ros2_kart_racing#15] [General]
If possible, please provide a way to test that everything has been set up and our local installation of ROS2 was successful.

Thanks,
DS

**Comment by gstavrinos** [comment]
Maybe it wasn't clear enough on the slides, and appeared `ros2ez` specific, but you can just try running a `ros2 topic list` command. You should see two topics even if nothing is running, but not encountering an error during execution should be enough for the next steps!

George

**Comment by dSofikitis** [comment]
Great, thanks.

**Comment by gstavrinos** [comment]
I added an announcement regarding the use of the discussions, since your question raised a problem that I did not know of: Resolved non-Q&A threads are closed and get filtered out with the default discussions filter. I hope you are ok with using this question as an example!

**Comment by dSofikitis** [comment]
No problem. I didn't know about it either.
